modules/ipi-install-preparing-the-bare-metal-node.adoc

```diff
@@ -91,7 +77,7 @@ If the MAC address of an existing bare metal node matches the MAC address of a b
 +
 [source,bash]
 ----
-[kni@provisioner ~]$ oc -n openshift-machine-api create -f bmh.yaml
```

Comment:
Per the Supplementary Style Guide, [user-replaced values](https://redhat-documentation.github.io/supplementary-style-guide/#user-replaced-values) within code blocks need the [subs="+quotes"] attribute to apply text formatting.